### PR TITLE
[EVENTVWR] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Improvements based on recent Romanian translation document revision 06e89b2.

This files was not translated in Romanian language in any Win version (neither XP/2k3+, nor in the previous ones). I tried to translate this file as close as possible in the way I did in the previous PRs.

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: base/applications/mscutils/eventvwr/lang/ro-RO.rc [hunk 1]
```diff
@@ -115,79 +115,79 @@ END
...
 END
 
 STRINGTABLE
 BEGIN
 /* Please note that explicit TAB characters '\t' are used to fix the alignment of the message in the MessageBox */
-    IDS_USAGE "Inspector de Evenimente ReactOS\n\
+    IDS_USAGE "Vizualizatorul de evenimente ReactOS\n\
 \n\
-EventVwr [nume calculator] [/L:<fișier jurnal de evenimente>] [/?]\n\
+EventVwr [numele calculatorului] [/L:<fișier de jurnal de evenimente>] [/?]\n\
```
Comment (JoachimHenze) [inline base/applications/mscutils/eventvwr/lang/ro-RO.rc:179]: In other places, e.g. SHELL32 you structurally changed from "calculator" to "computer". Why not in this module as well? (and in other locations/matches of this PR)

File: base/applications/mscutils/eventvwr/lang/ro-RO.rc [hunk 2]
```diff
@@ -229,12 +229,12 @@ ID eveniment:        %s\r\n\
 Dată:                %s\r\n\
 Oră:                 %s\r\n\
 Utilizator:          %s\r\n\
-Calculator:          %s\r\n\
+Computer:            %s\r\n\
```
Comment (JoachimHenze) [inline base/applications/mscutils/eventvwr/lang/ro-RO.rc:232]: E.g. Here you call it Computer. Which I do think is the better translation indeed.

File: base/applications/mscutils/eventvwr/lang/ro-RO.rc (hunk 1 above)
Comment (AndreiMiloiu) [reply base/applications/mscutils/eventvwr/lang/ro-RO.rc]: Fixed! I didn't pay attention.

**Pull request merged**